src/vignette/storage/static_assets.clj

```diff
@@ -7,7 +7,7 @@
 (defn- parse-content-disp
   [header]
   (when-let [m (if header (re-find #"(?i)filename=\"(.*)\"" header))]
-    (second m)))
+    (or (second m) "")))
```

Comment:
My nawias knowledge is very limited, but I'm assuming that will still create this field with "" Not sure if that's on purpose or we should just skip this field.